terraform/modules/custom-domains/main.tf

```diff
@@ -0,0 +1,24 @@
+
+module "primary-region" {
+  source = "../certificate-and-domain"
+
+  api_name              = var.api_name
+  certificate_subdomain = var.certificate_subdomain
+  cloudflare_zone_name  = var.cloudflare_zone_name
+
+  providers = { aws = aws }
```

Comment:
This line is optional since you're passing the default to the default.

Reply:
I think I'll leave it (unless you feel strongly otherwise) to help keep the differences between these two module usages all explicit (rather than having to know the defaults). I feel like it helps make it obvious what the differences are between the two `module` blocks in this file.